NearbyConnectionsWalkieTalkie/app/build.gradle

```diff
@@ -27,8 +27,8 @@ android {
 dependencies {
     implementation 'com.google.android.gms:play-services-nearby:16.0.0'
```

Comment:
nit: please update to the latest version - 18.0.1

Reply:
Another commit fix this, thanks